lib/galaxy/authnz/custos_authnz.py

```diff
@@ -42,6 +42,9 @@ def __init__(self, provider, oidc_config, oidc_backend_config, idphint=None):
         elif provider == 'keycloak':
             self._load_config_for_keycloak()
 
+    def _decode_token(self, token):
```

Comment:
For the purpose of anyone accidentally calling this when they meant to verify the signature, can you push `verify_signature` to the function arguments and default it to True ? Or rename it to `_decode_token_no_verify_signature` ? This seems like something we need to be explicit about.

Reply:
It should only be for internal usage here in the custos module, which only works the one way, but I can rename it.  Now that I'm thinking about it though, I'm going to revisit why we can't just validate the signature prior to proceeding here.